lib/themes/dosomething/paraneue_dosomething/js/app.js

```diff
@@ -78,11 +79,12 @@ $(document).ready(function() {
   ShowMore.init();
 
   // Intialize takeover modal.
-  Takeover.init();
+  // Takeover.init();
 
   // Add fallback for HTML5 video on mobile tiles
   Tile.init();
 
+  let donation = new OrganDonation($('.takeover-container'));
```

Comment:
Does mean we'll try to instantiate the `OrganDonation` takeover on every page now? If so, it might be worth exposing an `init` function from this module like some of the others do that checks if we're on the right page first.